WcaOnRails/spec/rails_helper.rb

```diff
@@ -34,6 +34,9 @@
 Capybara.javascript_driver = :poltergeist
 
 RSpec.configure do |config|
+  # For test files
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
```

Comment:
As already mentioned, I'd love to avoid bringing fixtures into our codebase. See http://www.danielstjules.com/2013/09/11/testing-file-uploads-in-rails-without-fixtures/ for an example of how file uploads can be tested without using fixtures.

Reply:
Can we get rid of this now?